src/AppInstallerRepositoryCore/Microsoft/SQLiteIndexSource.cpp

```diff
@@ -124,21 +179,19 @@ namespace AppInstaller::Repository::Microsoft
 
             std::shared_ptr<IPackageVersion> GetLatestAvailableVersion() const override
             {
-                // Although we could potentially increase efficiency here, this should be fine.
-                std::vector<PackageVersionKey> versions = GetAvailableVersionKeys();
-
-                if (!versions.empty())
-                {
-                    return GetAvailableVersion(versions[0]);
-                }
-
-                return {};
+                return GetLatestVersionInternal();

```

Comment:
>GetLatestVersionInternal() [](start = 23, length = 26)

Maybe PackageBase::GetLatestVersionInternal() to be explicit and match style in GetProperty above